corehq/apps/receiverwrapper/forms.py

```diff
@@ -18,6 +24,22 @@ def __init__(self, *args, **kwargs):
                 choices=self.formats,
             )
 
+        self.helper = FormHelper(self)
```

Comment:
while this is a nice shortcut, it's not as readable to someone revisiting this code later on. It would be much more straightforward to define the fields explicitly in a layout object and wrap everything in a fieldset there, and prepend the text inline with the actual field